KalturaMediaServer/src/main/java/com/kaltura/media/server/managers/KalturaLiveManager.java

```diff
@@ -127,7 +128,7 @@ public boolean removeReferrer(ILiveEntryReferrer obj) {
 		}
 
 		public LiveEntryCache(KalturaLiveEntry liveEntry, KalturaMediaServerIndex serverIndex, String applicationName) {
-			this(liveEntry);
+			// Check if this constructor is even relevant? Might be possible to delete?
```

Comment:
there is no base class for LiveEntryCache.
the this(liveEntry) that was removed is a bug, since it wont be passes to the other ctor (line 135), meaning the liveEntry member won't be initilized